source/process/marketing-guidelines.rst

```diff
@@ -0,0 +1,65 @@
+============================================================
+Mattermost Marketing Content Guidelines
+============================================================
+
+How to write purposefully and persuasively 
+==============================================================
+
+In this document we set out some easy-to-follow guidelines about writing. Follow them, while also relying on your own judgment. Writing for blogs, newsletters, magazine articles and other forms of content can then become a joy rather than a chore. 
+
+Before we get into the guidelines themselves, let's consider the goal and objectives (why we think written communication is important in the first place) and a few principles (things to bear in mind when pursuing those goals).
+
+Goal
+---------------------------------------------------------
+
+We write marketing content in order to encourage organizations to bring the benefits of modern communication with Mattermost to our target audience.
+
+Objectives
+---------------------------------------------------------
+
+In order to achieve this overriding goal, we must:
+
+ 1. Increase awareness of Mattermost as a solution to key challenges facing organizations as measured by new visits to mattermost.com
+ 2. Reduce the time for organizations to decide on deploying open source and commercial versions through educational content on product value 
+ 3. Increase the number of organizations deploying Mattermost swiftly, efficiently and effectively 
+ 4. Increase the compellingness and trustworthiness of Mattermost product and marketing content as measured by increase in return visits 
+
+The biggest uplift to inbound marketing will come from being featured in online publications such as The Hacker News.
+
+Principles
+---------------------------------------------------------
+
+In order to achieve these goals, our writing must be:
+
+ 1. Magnetic to IT. We need headlines and openings that draw in IT audiences (these include Linux admins, security experts, directors of end-user computing, CIOs). Bear in mind that, although they work in IT, these audiences are typically more interested in the business benefits of technology than technical features. 
```

Comment:
For Principles and Guidelines, propose having first sentence in bold just to have it called out more clearly.